### PR TITLE
grep for pixelx/y = 0 varedits

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request

Atomizing https://github.com/tgstation/tgstation/pull/54604

Dont make bad varedits

### CODE REVIEW

**Comment by Jared-Fogle** [comment]
> Not everyone uses strongdmm and seeing as I had to remove vars it doesnt work as advertised.If you need a snowflake make a subtype for it instead of bad varedits such as

I think it was oranges that said the types are not intended to be a dictionary of all items.

**Comment by Jared-Fogle** [comment]
image

**Comment by TiviPlus** [comment]
99.9% of pixelx/y varedits are just duplicating whats already set in the code for literally no reason and have nothing to do with being a dictionary
If your super snowflake offset object somehow needs offsets without looking bad then make a new one with 0 offset or use 1 offset

**Review by ShizCalev — DISMISSED** [review]
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#mapping-standards

image

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 1]
```diff
@@ -76413,9 +76413,7 @@
 /area/maintenance/starboard)
 "tgQ" = (
 /obj/structure/water_source/puddle,
-/obj/structure/flora/junglebush/large{
-	pixel_y = 0
```
Comment (ShizCalev) [inline _maps/map_files/MetaStation/MetaStation.dmm:76417]: this actually wasn't wrong. set it to 1 or something

File: _maps/map_files/PubbyStation/PubbyStation.dmm [hunk 2]
```diff
@@ -29645,9 +29643,7 @@
 /obj/structure/window/reinforced/spawner/east,
 /obj/structure/window/reinforced/spawner,
 /obj/structure/flora/grass/jungle/b,
-/obj/structure/flora/junglebush/large{
-	pixel_y = 0
```
Comment (ShizCalev) [inline _maps/map_files/PubbyStation/PubbyStation.dmm:29649]: same.

these are a lil snowflake since they're 64x64

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: _maps/map_files/MetaStation/MetaStation.dmm (hunk 1 above)
Comment (Fikou) [reply _maps/map_files/MetaStation/MetaStation.dmm:76417]: lol

**Comment by Jared-Fogle** [comment]
@EOBGames you're the maptainer, do you want this

**Comment by EOBGames** [comment]
> @EOBGames you're the maptainer, do you want this

Provided all it does is sanitize variables, it's fine by me. Ultimately most should be handled automatically provided people are using SDMM (which they should), so a thing to catch stuff that slips through the net is useful.

**Comment by Jared-Fogle** [comment]
works for me

**Review by Mothblocks — DISMISSED** [review]

**Comment by Jared-Fogle** [comment]
@TiviPlus fix conflicts and do shiz's comments please thank you

**Pull request merged**